### PR TITLE
[DUOS-1520][risk=no] Add PR/Closeout fields

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
https://broadworkbench.atlassian.net/browse/DUOS-1520

This PR adds new fields to the DAR Data object to support Progress Reports and Closeouts.

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/java/org/broadinstitute/consent/http/models/Presentation.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package org.broadinstitute.consent.http.models;
+
+public record Presentation(String title, String link, String date, String authors, String citation) {
```
Comment (connorlbark) [inline src/main/java/org/broadinstitute/consent/http/models/Presentation.java]: Should `authors` be an array? Should date be a `Date` object, or link be `URI`? Not sure if whatever parsing library we're using supports those objects

File: src/main/java/org/broadinstitute/consent/http/models/Publication.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package org.broadinstitute.consent.http.models;
+
+public record Publication(String title, String pubmedId, String date, String authors, String citation) {
```
Comment (connorlbark) [inline src/main/java/org/broadinstitute/consent/http/models/Publication.java]: Same thoughts here

**Review by connorlbark — APPROVED** [review]
Looks good, matches what is expected according to the ticket!

**Review by kaitlynoflaherty — APPROVED** [review]
Besides Connor's questions, looks good!

File: src/main/java/org/broadinstitute/consent/http/models/Presentation.java (hunk 1 above)
Comment (rushtong) [reply src/main/java/org/broadinstitute/consent/http/models/Presentation.java]: I'm hesitant to model these any more granularly than generic strings. The current mocks are super simple. For example, this is a presentation:
Screenshot 2023-05-02 at 12 15 48 PM

And this is a publication:
Screenshot 2023-05-02 at 12 16 33 PM

However, looking at this again, I think citation needs to be a boolean, not a string. I think the earlier versions were not booleans but looking at the mocks again, they are.

**Pull request merged**